tests/test_basic.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+
+from patchwork.actions import AskSubquestion, Reply, Unlock
+from patchwork.datastore import Datastore
+from patchwork.scheduling import RootQuestionSession, Scheduler
+
+
+# Note: This test is hard to read and tightly coupled with the implementation.
+# Over the next week I will make refactorings and implement infrastructure to
+# make it easy to write better tests. Or write more fine-grained unit tests
+# instead. I haven't decided yet. (RM 2018-06-09)
```

Comment:
Remove the comment above? This sort of thing seems mostly useful as a comment on the PR, not as part of the codebase.

Reply:
Good point.